webpack/assets/javascripts/react_app/components/common/forms/FormAutocomplete/index.js

```diff
@@ -0,0 +1,29 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as actions from '../../../../components/AutoComplete/AutoCompleteActions';
+// import reducer from '../../../../components/AutoComplete/AutoCompleteReducer';
+import FormAutocomplete from './FormAutocomplete';
+
+const mapStateToProps = ({
+  autocomplete: {
+    error, results, searchQuery, status, url, controller, trigger, isDisabled,
+  },
+}) => ({
+  error,
+  results,
```

Comment:
I wonder if it make sense to use a memorization on the results, though it seems that only one autocomplete object is actually stored...

Reply:
it might be a great follow up PR !!